.travis.yml

```diff
@@ -1,24 +1,24 @@
 os:
   - linux
-dist: trusty
+  - osx
 
 language: c
 
+env:
+  - BRANCH=0.19.4
+  - BRANCH=devel
+
 cache:
   directories:
-    - "$HOME/.nimble"
-    - "$HOME/.choosenim"
+    - "$HOME/.choosenim/toolchains/nim-0.19.4"
```

Comment:
There's nothing worth caching in ~/.nimble. Also, caching only stable since the binary won't change. Devel shouldn't be cached so that we always test with latest. Otherwise we are testing with an old devel after first cache.

Reply:
> There's nothing worth caching in ~/.nimble. 

There is.

The `.nimble/bin` is added to the PATH below, if this directory doesn't get cached then failures will happen.

Reply:
@genotrance as far as I'm concerned you need to revert this because it will break things, please do so.